R/geom_stripped_rows.R

```diff
@@ -0,0 +1,178 @@
+#' Alternating Background Colour
+#'
+#' Add alternating background color along the y-axis. The geom takes default
+#' aesthetics \code{odd} and \code{even} that receive color codes.
+#'
+#' @inheritParams ggplot2::layer
+#' @inheritParams ggplot2::geom_rect
+#' @param xfrom,xto limitation of the strips along the x-axis
```

Comment:
`xmin` and `xmax` are used by errorbars and therefore it creates bad interaction with stripped rows. In such case, users should unmap these two aesthetics when calling `geom_stripped_rows()` and manually define them as `-Inf` and `Inf`. 

`xfrom` and `xto` are more parameters/arguments of the geom rather than aesthetics. It's, therefore, safer to use other terms.

Reply:
Cool. Thank you!